src/wallet/rpcwallet.cpp

```diff
@@ -401,11 +401,20 @@ UniValue SendMoney(CWallet* const pwallet, const CCoinControl &coin_control, std
     int nChangePosRet = -1;
     bilingual_str error;
     CTransactionRef tx;
-    bool fCreated = pwallet->CreateTransaction(recipients, tx, nFeeRequired, nChangePosRet, error, coin_control, !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
+    UniValue entry(UniValue::VOBJ);
+    std::string feeReason;
+    bool fCreated = pwallet->CreateTransaction(recipients, tx, nFeeRequired, nChangePosRet, error, coin_control, feeReason, !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
     if (!fCreated) {
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, error.original);
     }
     pwallet->CommitTransaction(tx, std::move(map_value), {} /* orderForm */);
+
+    if(verbose){
+        entry.pushKV("hex", tx->GetHash().GetHex());
+        entry.pushKV("Fee Reason", feeReason);  
```

Comment:
I don't think we put spaces as the key in a json object as spaces tend to break things. Typically we do lowercase snake_case.